src/reynir_correct/annotation.py

```diff
@@ -125,3 +128,15 @@ def suggest(self) -> Optional[str]:
         """ A suggested correction for the token span, as a text string
             containing tokens delimited by spaces """
         return self._suggest
+
+
+    @property
+    def original(self) -> Optional[str]:
+        """ The original text for the error """
+        return self._original
+
+    @property
+    def correction(self) -> Optional[str]:
+        """ A suggested correction for the token span, as a text string
+            containing tokens delimited by spaces """
+        return self._correction
```

Comment:
Where is this property coming from? Doesn't seem to be assigned anywhere.

Reply:
Leftovers from experiments, merged it with self._suggest